src/math/OrthoNormalBasis.js

```diff
@@ -148,25 +150,24 @@ OrthoNormalBasis.Z0Plane = function () {
 }
 
 OrthoNormalBasis.prototype = {
+
   getProjectionMatrix: function () {
```

Comment:
see my comments above about potentially moving these out to just functions of mat4, and bby orthoBasis !